contracts/teller-network/OfferStore.sol

```diff
@@ -164,6 +181,7 @@ contract OfferStore is Stakable, SecuredFunctions, Proxiable {
         uint256 offerId = offers.push(newOffer) - 1;
         offerWhitelist[msg.sender][offerId] = true;
         addressToOffers[msg.sender].push(offerId);
+        offerCnt[msg.sender]++;
```

Comment:
I guess when `offerCnt[msg.sender]` is undefined, it starts at 0?

Reply:
yes